bin/analyzeShearTest.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2015 AURA/LSST
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+import argparse
+import math
+import os.path
+
+import lsst.daf.persistence as dafPersist
+import lsst.afw.table as afwTable
+"""
+analyzeShearTest is a portion of the shear test suite which summarizes the information
+from all of the GalSim runs and creates a table which is suitable for plotting.
+
+The data is divided into GalSim runs each with a filter[23], seeing[.05 .07 .09] and shear.
+All subfields in the run have the same shear, but each has a randomly chosen shear angle.
+In this program, we average each subfield to get an estimator of the shear value.  In our
+initial tests, there are 32 subfields with a different shear angle.  There are 1024 galaxies
+for each subfield, comprised of 512 galaxy pair rotated at 90 degrees to each other.
+
+These 32 samples are averaged to provide an overall estimator of the shear, and the stddev of
+these samples provides an error for this estimator.
+"""
```

Comment:
Put module level docstrings like this one after the copyright but before any import statements (if they come after the import statement, they won't actually get used as docstrings).  Also put a blank line between the docstring and the import statements, and between the imports and the code that follows.